### PR TITLE
add mesh voxelizer

### PR DESCRIPTION
**Pull request by hadjiszs**
# PR Description

CLI mesh voxelizer

c.f. DGtal-team/DGtal#1209

# Checklist
- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug `cmake` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: converters/mesh2vol.cpp [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+/**
+ * @file mesh2vol.cpp
+ * @ingroup converters
+ *
+ * @date 2016/09/28
```
Comment (kerautret) [inline converters/mesh2vol.cpp]: are you sure for the date ;)

File: converters/mesh2vol.cpp [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+ @brief Convert a mesh file into a 26-separated or 6-separated voxelization in a given resolution grid.
+
+@b Usage: mesh2vol [input]
+
+@b Allowed @b options @b are:
+
+@code
+  -h [ --help ]                   display this message
+  -i [ --input ] arg              mesh file (.off)
+  -o [ --output ] arg             filename of ouput voxelization file (.off)
```
Comment (kerautret) [inline converters/mesh2vol.cpp]: .off => .vol and precise volumetric file like in source code ;)

File: converters/mesh2vol.cpp [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+int main( int argc, char** argv )
+{ 
+  // parse command line ----------------------------------------------
+  po::options_description general_opt("\nAllowed options are");
+  general_opt.add_options()
+    ("help,h", "display this message")
+    ("input,i", po::value<std::string>(), "mesh file (.off) " )
+    ("output,o", po::value<std::string>(), "filename of ouput volumetric file (Vol,PGM3D,..).")
```
Comment (kerautret) [inline converters/mesh2vol.cpp]: vol, pgm3d (lower case)

**Comment by dcoeurjo** [comment]
@kerautret can I (or you) merge this one ?

**Comment by kerautret** [comment]
yes merging (I add just Monir Hadji as authors afterward on the master)

**Comment by dcoeurjo** [comment]
👌 thx

**Pull request merged**